website/docs/d/workspace_ids.html.markdown

```diff
@@ -23,20 +28,20 @@ data "tfe_workspace" "test" {
 
 The following arguments are supported:
 
-* `names` - (Required) A list of workspace names.
-* `organization` - (Required) Name of the organization.
+* `names` - (Required) A list of workspace names to search for. Names that don't
+  match a real workspace will be omitted from the results, but are not an error.
 
-~> The list of names can be used to search for workspaces with matching names.
-  Additionally you can also use a single entry with a wildcard (e.g. `"*"`) which
-  will match all names. Using a partial string together with a wildcard (e.g.
-  `"my-workspace-*"`) is **not** supported.
+    To select _all_ workspaces for an organization, provide a list with a single
```

Comment:
Because `["*"]` isn't really a "special case." It's just that the `names` attribute accepts two different kinds of values: either a list of names, or a list of one asterisk. 

Whenever possible, the bullet point for an attribute should fully describe its allowed values, and putting the `~>` below the attribute list split that info into two different places.

Reply:
Check... I took the approach from here: https://www.terraform.io/docs/configuration/resources.html#ignore_changes

But I guess this is fine as well. Yet if we just add this as additional text, should we then still include the blank line? Of just write it as one block?

Reply:
Ah, gotcha! Well, for what it's worth, it looks like we [changed that for the 0.12 docs](https://github.com/hashicorp/terraform/blob/master/website/docs/configuration/resources.html.md#lifecycle-lifecycle-customizations). 

I tried a few ways to get it into a single paragraph, but they all made it harder to read and harder to skim. Compare:

> - `names` - (Required) Either a list of workspace names to search for, or a list with a single asterisk (`["*"]`) to request all workspaces. (No other use of wildcards is supported.) Names that don't match a real workspace will be omitted from the results, but are not an error.

Reply:
Check 👍